pkg/sqs/sqs.go

```diff
@@ -29,10 +29,9 @@ import (
 )
 
 const (
-	approximateNumberOfMessages           = "ApproximateNumberOfMessages"
-	approximateNumberOfMessagesNotVisible = "ApproximateNumberOfMessagesNotVisible"
-	maxNumberOfMessages                   = 10 // maxNumberOfMessages is messages to return From SQS Valid values: 1 to 10. Default: 1
-	waitTimeSeconds                       = 20 // waitTimeSeconds specifies the time (in seconds) the call waits for a message to arrive in the queue before returning. If a message arrives, the call returns early; if not and the time expires, it returns an empty message list.
+	approximateNumberOfMessages = "ApproximateNumberOfMessages"
+	maxNumberOfMessages         = 10 // maxNumberOfMessages is messages to return From SQS Valid values: 1 to 10. Default: 1
+	waitTimeSeconds             = 20 // waitTimeSeconds specifies the time (in seconds) the call waits for a message to arrive in the queue before returning. If a message arrives, the call returns early; if not and the time expires, it returns an empty message list.
```

Comment:
I remember we have a `timeout` passed from the numa container, shall we use that? @KeranYang

Reply:
Yes. Because the Read function here only contains single sqsClient receive call, we can use the timeout passed in to configure the sqs API call directly.

Reply:
As per aws docs - WaitTimeSeconds  must be <= 0 and >= 20 if provided. References -https://docs.aws.amazon.com/powershell/latest/reference/?page=New-SQSQueue.html&tocid=New-SQSQueue
<img width="1029" alt="Screenshot 2023-12-18 at 12 40 09 PM" src="https://github.com/numaproj-contrib/aws-sqs-source-go/assets/25385196/9fbb998b-fbf8-4661-b813-ca5eb41564eb">